### PR TITLE
Add missing redirect for java extensions page

### PR DESCRIPTION
**Pull request by svrnm**

### CODE REVIEW

**Review by mateuszrzeszutek — APPROVED** [review]

File: content/en/docs/instrumentation/java/automatic/extensions.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ---
 title: Extensions
+aliases:
+  - /docs/instrumentation/java/extensions/
```
Comment (chalin) [inline content/en/docs/instrumentation/java/automatic/extensions.md:4]:
```suggestion
aliases: [/docs/instrumentation/java/extensions]
```

**Comment by chalin** [comment]
- For the record, this is a followup to #2690
- Preview: https://deploy-preview-2695--opentelemetry.netlify.app/docs/instrumentation/java/automatic/extensions/
- Redirect test: https://deploy-preview-2695--opentelemetry.netlify.app/docs/instrumentation/java/extensions/

File: content/en/docs/instrumentation/java/automatic/extensions.md (hunk 1 above)
Comment (chalin) [inline content/en/docs/instrumentation/java/automatic/extensions.md:4]: Nit: when there's a single alias, I prefer the compact one-line form (and we can drop the trailing `/`):

```suggestion
aliases: [/docs/instrumentation/java/extensions]
```

**Review by chalin — APPROVED** [review]
Otherwise, LGTM, thanks!

**Review by cartermp — APPROVED** [review]

**Pull request merged**